app/basicComponents/Input.tsx

```diff
@@ -46,12 +46,12 @@ const ExtraTxt = styled.div`
 `;
 
 type Props = {
-  onChange?: ({ value }: { value: string }) => void;
-  onChangeDebounced?: ({ value }: { value: string }) => void;
+  onChange?: ({ value }: { value: any }) => void;
```

Comment:
🤔 
Why `any`? 
Actually, `input` always deals with string. So what is the point?

Reply:
reverted it